index.js

```diff
@@ -48,7 +48,7 @@ const pRetry = (input, options) => new Promise((resolve, reject) => {
 			if (error instanceof AbortError) {
 				operation.stop();
 				reject(error.originalError);
-			} else if (error instanceof TypeError) {
+			} else if (error instanceof TypeError && error.message !== 'Failed to fetch') {
```

Comment:
Shouldn't we lax this condition a bit? Checking the error message for an exact match is [very restrictive](https://github.com/whatwg/fetch/issues/526#issuecomment-554604080).

Reply:
Yeah I agree, matching the string is ugly, have a better idea?

Reply:
Well, you're right. I had something else in mind but it will just overcomplicate things. The issue is documented in the README, so I guess it's alright. 💪

Reply:
so whats the next step? :)

Reply:
@sindresorhus will come back to you soon